addons/website/static/src/js/menu/navbar.js

```diff
@@ -37,6 +38,12 @@ var WebsiteNavbar = publicWidget.RootWidget.extend({
      */
     start: function () {
         var self = this;
+            dom.initAutoMoreMenu(this.$('ul.o_menu_sections'), {
+                maxWidth : function () {
```

Comment:
Remove the extra space :) Your linter should have told it to you :) https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines